docs/root/configuration/http_filters/lua_filter.rst

```diff
@@ -312,6 +312,57 @@ Returns the current request's underlying :repo:`connection <include/envoy/networ
 
 Returns a :ref:`connection object <config_http_filters_lua_connection_wrapper>`.
 
+importPublicKey()
+^^^^^^^^^^^^^^^^^
+
+.. code-block:: lua
+  
+  pkey = handle:importPublicKey(keyder, keyderLenght)
+
+Returns public key which is used by :ref:`verifySignature <verify_signature>` to verify digital signature. 
+
+.. attention::
+
+  Remember to call :ref:`releasePublicKey <release_public_key>` when *pkey* is not needed to avoid memory leak.
+
+.. _release_public_key:
+
+releasePublicKey()
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: lua
+  
+  handle:releasePublicKey(pkey)
+
+Free the resource of *pkey*
+
+decodeBase64()
+^^^^^^^^^^^^^^
+
+.. code-block:: lua
+  
+  raw = request_handle:decodeBase64(str)
+
+Decodes base64 encoded string. Returns *nil* if input string is invalid.
+
+.. _verify_signature:
+
+verifySignature()
+^^^^^^^^^^^^^^^^^
+
+.. code-block:: lua
+
+  ok, error = verifySignature(pkey, hashFunction, signature, signatureLength, data, dataLength)
+
+Verify signature using provided parameters. *pkey* is the public key, *hashFunction* is the variable 
+for hash function which be used for verifying signature. *MD4*, *MD5*, *SHA1*, *SHA224*, *SHA256*, 
+*SHA384*, *SHA512* and *MD5_SHA1* are supported. *signature* is the signature to be verified. 
```

Comment:
Some of those hash functions shouldn't be used anymore... Could you drop MD4, MD5 and MD5_SHA1?